ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Reporter.scala

```diff
@@ -102,14 +102,19 @@ object Reporter {
           }
       }
 
-    override def report(results: Vector[LedgerTestSummary]): Unit = {
+    override def report(results: Vector[LedgerTestSummary], testsSeed: String): Unit = {
       s.println()
       s.println(blue("#" * 80))
       s.println(blue("#"))
       s.println(blue("# TEST REPORT"))
       s.println(blue("#"))
       s.println(blue("#" * 80))
 
+      s.println()
+      s.println(yellow("### CONFIG"))
+      s.println()
+      s.println(cyan(s"Tests seed = $testsSeed"))
```

Comment:
Please use a single name to refer to this information. The original `**identifierSuffix` seems to be a sensible choice.

Furthermore, this is really not a seed.

Reply:
I don't have context how it's used inside the tests. I used name convention from jira ticket. Fixed.